rest/v9/channel.ts

```diff
@@ -529,6 +529,24 @@ export interface RESTPostAPIChannelThreadsJSONBody {
 /**
  * https://discord.com/developers/docs/resources/channel#start-thread-with-message
  */
+export type RESTPostAPIChannelMessagesThreadsResult = APIChannel;
+
+/**
+ * https://discord.com/developers/docs/resources/channel#start-thread-without-message
+ */
+export interface RESTPostAPIChannelThreadsJSONBody extends RESTPostAPIChannelMessagesThreadsJSONBody {
+	/**
+	 * The type of thread to create
+	 */
+	type: Extract<
+		ChannelType,
+		ChannelType.GuildNewsThread | ChannelType.GuildPublicThread | ChannelType.GuildPrivateThread
+	>;
```

Comment:
There's no need to overcomplicate it

```suggestion
	type: ChannelType.GuildNewsThread | ChannelType.GuildPublicThread | ChannelType.GuildPrivateThread;
```